src/ImageRunModal.jsx

```diff
@@ -328,104 +331,124 @@ export class ImageRunModal extends React.Component {
-                </FormGroup>
-
-                <FormGroup fieldId='run-image-dialog-command' label={_("Command")}>
-                    <TextInput id='run-image-dialog-command'
-                               placeholder={_("Command")}
-                               value={dialogValues.command || ''}
-                               onChange={value => this.onValueChanged('command', value)} />
-                </FormGroup>
-
-                <FormGroup fieldId='run-image-dialog-memory' label={_("Memory limit")}>
-                    <InputGroup className="ct-input-group-spacer-sm" id="run-image-dialog-memory-limit">
-                        <Checkbox id="run-image-dialog-memory-limit-checkbox"
+                        </FormGroup>
+                    </FlexItem>
+                </Flex>
+                <Tabs activeKey={activeTabKey} onSelect={this.handleTabClick}>
+                    <Tab eventKey={0} title={<TabTitleText>Details</TabTitleText>} className="pf-l-grid pf-m-gutter">
+                        <FormGroup fieldId='run-image-dialog-image' label={_("Image")} hasNoPaddingTop>
+                            <div id='run-image-dialog-image'> { image.RepoTags ? image.RepoTags[0] : "" } </div>
+                        </FormGroup>
+
+                        <FormGroup fieldId='run-image-dialog-command' label={_("Command")}>
+                            <TextInput id='run-image-dialog-command'
+                           placeholder={_("Command")}
+                           value={dialogValues.command || ''}
+                           onChange={value => this.onValueChanged('command', value)} />
+                        </FormGroup>
+
+                        <FormGroup fieldId="run=image-dialog-tty">
+                            <Checkbox id="run-image-dialog-tty"
+                              isChecked={this.state.hasTTY}
+                              label={_("With terminal")}
+                              onChange={checked => this.onValueChanged('hasTTY', checked)} />
+                        </FormGroup>
+
+                        <FormGroup fieldId='run-image-dialog-memory' label={_("Memory limit")}>
+                            <InputGroup className="ct-input-group-spacer-sm" id="run-image-dialog-memory-limit">
+                                <Checkbox id="run-image-dialog-memory-limit-checkbox"
                                   isChecked={this.state.memoryConfigure}
                                   onChange={checked => this.onValueChanged('memoryConfigure', checked)} />
-                        <TextInput type='number'
+                                <TextInput type='number'
                                    value={dialogValues.memory}
                                    id="run-image-dialog-memory"
                                    step={1}
                                    min={0}
                                    isReadOnly={!this.state.memoryConfigure}
```

Comment:
This needs to be discussed with @garrett and adjusted. When disabled the memory limit is unset, having this field disabled is misleading.

Reply:
I agree that it's misleading but I am not sure what the new UX should be, should we make an improvement issue for this?